app/email_utils.py

```diff
@@ -1427,3 +1433,18 @@ def save_email_for_debugging(msg: Message, file_name_prefix=None) -> str:
         return file_name
 
     return ""
+
+
+def get_dmarc_status(msg: Message) -> Optional[DmarcCheckResult]:
+    spam_result = msg.get_all(headers.SPAMD_RESULT)
+    if not spam_result:
+        return None
+    spam_entries = [entry.strip() for entry in spam_result[-1].split("\n")]
+    for iPos in range(len(spam_entries)):
```

Comment:
nits: should we use underscore instead of camel case for the variable name here?

Reply:
arghh.. it gets confusing changing between projects with camelCase and snake_case.. I'll change it :+1: